hack/generate-agent-config.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script helps configure Nephe cluster and generates necessary kubeconfigs for importing agented VMs.
+# command reference: https://github.com/antrea-io/antrea/blob/main/docs/external-node.md#prerequisites-on-kubernetes-cluster.
+
+set -e
+
+# Constants.
+EKS="eks"
+AKS="aks"
+CLUSTER_NAME=$(kubectl config current-context)
+K8S_KUBECONFIG="antrea-agent.kubeconfig"
+ANTREA_KUBECONFIG="antrea-agent.antrea.kubeconfig"
+NAMESPACE="vm-ns"
```

Comment:
Dont hard code Namespace and Service account. Take that as input to the script

Reply:
I want this script to be simple instead of having a lot of arguments, maybe we can make service account an optional argument? We discussed namespace in the previous comment